std/container/slist.d

```diff
@@ -589,6 +601,43 @@ Complexity: $(BIGOH n)
 
 /// ditto
     alias stableLinearRemove = linearRemove;
+
+/**
+Removes the first occurence of an element from the list in linear time.
+
+Params:
+    value = value of the node to be removed
+
+Complexity: $(BIGOH n)
+     */
+    void linearRemoveElement(T value)
```

Comment:
I think it would be better to return a boolean value - `true` iff the element existed and was successfully removed (`false` otherwise).

Reply:
In which case, `findNodeByValue` should probably return `null` (without asserting) if `_root` is also `null`.